### PR TITLE
Titanic_ Survival _Prediction

### LINKED ISSUE #44: Titanic - Machine Learning from Disaster
## About this file

- These predictions assume only female passengers survive.

### The data has been split into two groups: 

- training set (train.csv)
- test set (test.csv)

### Dataset

[https://www.kaggle.com/c/titanic/data](url)

### PR DESCRIPTION
**Pull request by harshita9621**
fixes #44 

# About the data

- Predict survival on the Titanic and get familiar with ML basics.

### Accuracy of different models used

By using Logistic Regression I got 
 ```python
    test accuracy score of  Logistic Regression =  0.8188755470766468
 ``` 

By using Random Forest Classifier I got 
 ```python
    test accuracy score of Random Forest =  0.8340807174887892
 ``` 
 
 By using Decision Tree Classifier I got 
 ```python
    test accuracy score of Decision Tree =  0.8071748878923767
 ``` 
 
  By using  Support Vector Classifier I got 
 ```python
    test accuracy score of Support Vector Classifier =  0.8116591928251121
 ``` 
 
  By using GradientBoostingClassifier  I got 
 ```python
    test accuracy score of GradientBoostingClassifier =  0.8430493273542601
 ``` 
 
  By using AdaBoostClassifier I got 
 ```python
    test accuracy score of  AdaBoostClassifier = 0.8251121076233184
 ``` 
 
  By using KNN I got 
 ```python
    test accuracy score of KNN =  0.8071748878923767
 ``` 
 
  By using VotingClassifier I got 
 ```python
    test accuracy score of VotingClassifier =  0.852
 ``` 
 
  By using SGDClassifier I got 
 ```python
    test accuracy score of SGDClassifier =  0.7847533632286996
 ```

### CODE REVIEW

**Comment by utkarshxy** [comment]
@harshita9621 You need to change the name of your folder
